src/Interpreters/InterpreterInsertQuery.h

```diff
@@ -73,12 +73,17 @@ class InterpreterInsertQuery : public IInterpreter, WithContext
 
     ASTPtr query_ptr;
     const bool allow_materialized;
-    const bool no_squash;
-    const bool no_destination;
+    bool no_squash = false;
+    bool no_destination = false;
     const bool async_insert;
 
     std::vector<std::unique_ptr<ReadBuffer>> owned_buffers;
 
+    std::pair<std::vector<Chain>, std::vector<Chain>> buildPreAndSyncChains(size_t presink_streams, size_t sink_streams, StoragePtr table, const StorageMetadataPtr & metadata_snapshot, const Block & query_sample_block);
```

Comment:
May be you wanted to name this method `buildPreAndSyncChains` -> `buildPreAndSinkChains`?

Reply:
Yes, indeed.